.github/workflows/model_performance.yml

```diff
@@ -0,0 +1,99 @@
+name: Model Performance Characteristics
```

Comment:
Nit: for the Actions, we have been doing the "what", "why", "when" comment sections so lets stick to that format

Reply:
pushed first draft of these comments.